products/expedition/docs/expedition_workflow_searchrule.md

```diff
@@ -0,0 +1,766 @@
+---
+id: expedition_workflow_searchrule
+title: List security rules contain specific subnet
```

Comment:
Should this be "List security rules containing a specific subnet" ?

Reply:
Sure , please go ahead to change the wording